Autopilot/Utilities.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data.SqlClient;
+using System.Windows;
+
+namespace Autopilot
+{
+    class Utilities
+    {
+        public string DBconnStrg = Properties.Settings.Default.AutopilotConnectionString; //zentraler DB-ConnectionStrg
+
+        public void SQLCmd(string SQLCmd_txt)
+        {
+            SqlConnection conn = new SqlConnection(DBconnStrg);
```

Comment:
Sowas braucht keine eigene Klasse, das kann auch als statisches Feld mit ans MainWindow. Dann wird da einmal die Verbindung zur Datenbank aufgebaut (im Konstruktor) und dann haben wir das. Das sollte sich auch leichter pflegen lassen.

Für die eigentliche Interaktion mit der Datenbank hätte ich wie gesagt auch auf das EntityFramework gesetzt. Das würde die Interaktion mit verschiedenen Datenbanken möglich und einfach machen. Und der Rest dann per LINQ to SQL